statsmodels/tsa/statespace/kalman_filter.py

```diff
@@ -7,7 +7,6 @@
 from __future__ import division, absolute_import, print_function
 
 from warnings import warn
-from contextlib import contextmanager
```

Comment:
+1 on avoiding the duplicate import.  I'd rather it be up here than below, but either way.